screens/VitaminsScreen.js

```diff
@@ -0,0 +1,88 @@
+// https://medium.com/react-native-development/how-to-use-the-flatlist-component-react-native-basics-92c482816fe6
+import React, { Component } from "react";
+import { View, Text, FlatList } from "react-native";
+import { SearchBar, List, ListItem } from "react-native-elements";
+import * as Papa from "papaparse";
+
+import VitaminParse from "./parsing/VitaminParse";
+
+export default class VitaminList extends Component {
+    static navigationOptions = {
+        title: "Movement Vitamins"
+    }
+    
+    constructor() {
+	super(...arguments);
+
+	this.state = {
+	    loading: false,
+	    data: [],
+	    page: 1,
+	    seed: 1,
+	    error: null,
+	    refreshing: false,
+	};
+
+    }
+
+    async getData() {
+        // Temporary
```

Comment:
We will, im guessing, soon split this off into its own component to send and retrieve HTTPS requests to the postgres database? Do we know yet how to accomplish that? links are appreciated.